libs/daohaus-connect-feature/project.json

```diff
@@ -5,14 +5,17 @@
   "tags": [],
   "targets": {
     "vite-build": {
-      "executor": "@nxext/vite:package",
+      "executor": "@daohaus/executors:vite-helper",
       "outputs": ["{options.outputPath}"],
       "options": {
         "outputPath": "dist/libs/daohaus-connect-feature",
         "frameworkConfigFile": "@nxext/react/plugins/vite",
         "configFile": "libs/daohaus-connect-feature/vite.config.ts",
         "entryFile": "src/index.ts",
-        "generatePackageJson": true
+        "updateBuildableProjectDepsInPackageJson": true,
+        "buildableProjectDepsInPackageJsonType": "dependencies",
+        "packageJson": "libs/daohaus-connect-feature/package.json",
+        "project": "libs/daohaus-connect-feature"
```

Comment:
here is the change where we add our exectutor from `tools/executor` --- is this naming ok? 

let me know if you think of a better one

Reply:
Might be better to name it @daohaus/nx-plugin as I think the package could contain both executors and generators